cdap-app-fabric/src/main/java/io/cdap/cdap/metadata/LineageHTTPHandler.java

```diff
@@ -72,6 +72,15 @@
     this.fieldLineageAdmin = fieldLineageAdmin;
   }
 
+  /**
+   * Get the lineage information about a dataset. This method does not give any information on field level lineage.
+   *
+   * @param startStr the start timestamp, in milliseconds, inclusive
```

Comment:
is it really in milliseconds? Or does it allow relative time like "now-1m"?

Reply:
Good catch, it supports "now", will update the doc